Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.cs

```diff
@@ -294,7 +338,7 @@ private static void Textbox_TextChanging(TextBox textbox, TextBoxTextChangingEve
                 {
                     var textboxInitialValue = textbox.Text;
                     textbox.Text = displayText;
-                    SetTextBoxValue(textboxInitialValue, textbox, mask, representationDictionary, placeHolderValue[0], 0);
+                    SetTextBoxValue(textboxInitialValue, textbox, escapedMask, escapedChars, representationDictionary, placeHolderValue[0], 0);
```

Comment:
The changes to this function are missing? Looks like there are other errors in the build log too.  What issues were you having building the project?

Reply:
@michael-hawker these are the build errors:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0518	Predefined type 'System.Void' is not defined or imported	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	19	Active
Error	CS0006	Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Notifications\bin\Debug\uap10.0\Microsoft.Toolkit.Uwp.Notifications.dll' could not be found	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\CSC	1	Active
Error	CS0518	Predefined type 'System.Void' is not defined or imported	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	16	Active
Error	CS0518	Predefined type 'System.Void' is not defined or imported	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	18	Active
Error	CS0518	Predefined type 'System.Void' is not defined or imported	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	19	Active
Error	CS0518	Predefined type 'System.Void' is not defined or imported	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	21	Active
Error	CS0518	Predefined type 'System.Void' is not defined or imported	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	21	Active
Error	CS0246	The type or namespace name 'IList<>' could not be found (are you missing a using directive or an assembly reference?)	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	23	Active
Error	CS0518	Predefined type 'System.Void' is not defined or imported	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	25	Active
Error	CS0518	Predefined type 'System.Void' is not defined or imported	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	27	Active
Error	CS1061	'ToastContent' does not contain a definition for 'GetXml' and no accessible extension method 'GetXml' accepting a first argument of type 'ToastContent' could be found (are you missing a using directive or an assembly reference?)	Microsoft.Toolkit.Uwp.Samples.BackgroundTasks	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\TestBackgroundTask.cs	35	Active
Error		The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	DifferencesGen			
Error		The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.SampleApp			
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit\bin\Debug\netstandard2.0\Microsoft.Toolkit.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Connectivity\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.Connectivity.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Parsers\bin\Debug\netstandard2.0\Microsoft.Toolkit.Parsers.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.DeveloperTools\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.DeveloperTools.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Services\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.Services.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Controls.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Notifications\bin\Debug\uap10.0\Microsoft.Toolkit.Uwp.Notifications.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Services\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Services.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Debug\Microsoft.Toolkit.Uwp.Input.GazeInteraction\Microsoft.Toolkit.Uwp.Input.GazeInteraction.winmd'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Graph\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Controls.Graph.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Animations\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Animations.dll'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks\bin\Debug\Microsoft.Toolkit.Uwp.Samples.BackgroundTasks.winmd'	Microsoft.Toolkit.Uwp.SampleApp	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.SampleApp\Microsoft.Toolkit.Uwp.SampleApp.csproj		
Error		The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design			
Error		The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.UI.Controls.Design			
Error		The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	UnitTests			
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.dll'	UnitTests	E:\Experimental\WindowsCommunityToolkit\UnitTests\UnitTests.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Parsers\bin\Debug\netstandard2.0\Microsoft.Toolkit.Parsers.dll'	UnitTests	E:\Experimental\WindowsCommunityToolkit\UnitTests\UnitTests.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit\bin\Debug\netstandard2.0\Microsoft.Toolkit.dll'	UnitTests	E:\Experimental\WindowsCommunityToolkit\UnitTests\UnitTests.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.dll'	UnitTests	E:\Experimental\WindowsCommunityToolkit\UnitTests\UnitTests.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Controls.dll'	UnitTests	E:\Experimental\WindowsCommunityToolkit\UnitTests\UnitTests.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Services\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Services.dll'	UnitTests	E:\Experimental\WindowsCommunityToolkit\UnitTests\UnitTests.csproj		
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Connectivity\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.Connectivity.dll'	UnitTests	E:\Experimental\WindowsCommunityToolkit\UnitTests\UnitTests.csproj		
Error		The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	UnitTests.Notifications.UWP			
Error		Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.Notifications\bin\Debug\uap10.0\Microsoft.Toolkit.Uwp.Notifications.dll'	UnitTests.Notifications.UWP	E:\Experimental\WindowsCommunityToolkit\UnitTests\UnitTests.Notifications.UWP\UnitTests.Notifications.UWP.csproj		
Error		The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	UnitTests.Notifications.WinRT			
Error	NU1201	Project Microsoft.Toolkit.Uwp.UI.Controls.DataGrid is not compatible with net452 (.NETFramework,Version=v4.5.2) / win-x64. Project Microsoft.Toolkit.Uwp.UI.Controls.DataGrid supports: uap10.0.16299 (UAP,Version=v10.0.16299)	Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design.csproj	1	
Error	NU1201	Project Microsoft.Toolkit.Uwp.UI.Controls.DataGrid is not compatible with net452 (.NETFramework,Version=v4.5.2) / win. Project Microsoft.Toolkit.Uwp.UI.Controls.DataGrid supports: uap10.0.16299 (UAP,Version=v10.0.16299)	Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design.csproj	1	
Error	NU1201	Project Microsoft.Toolkit.Uwp.UI.Controls.DataGrid is not compatible with net452 (.NETFramework,Version=v4.5.2) / win-x86. Project Microsoft.Toolkit.Uwp.UI.Controls.DataGrid supports: uap10.0.16299 (UAP,Version=v10.0.16299)	Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design.csproj	1	
Error	NU1201	Project Microsoft.Toolkit.Uwp.UI.Controls.DataGrid is not compatible with net452 (.NETFramework,Version=v4.5.2). Project Microsoft.Toolkit.Uwp.UI.Controls.DataGrid supports: uap10.0.16299 (UAP,Version=v10.0.16299)	Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design\Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.Design.csproj	1	
Error	NU1201	Project Microsoft.Toolkit.Uwp.UI.Controls is not compatible with net452 (.NETFramework,Version=v4.5.2) / win. Project Microsoft.Toolkit.Uwp.UI.Controls supports: uap10.0.16299 (UAP,Version=v10.0.16299)	Microsoft.Toolkit.Uwp.UI.Controls.Design	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Design\Microsoft.Toolkit.Uwp.UI.Controls.Design.csproj	1	
Error	NU1201	Project Microsoft.Toolkit.Uwp.UI.Controls is not compatible with net452 (.NETFramework,Version=v4.5.2). Project Microsoft.Toolkit.Uwp.UI.Controls supports: uap10.0.16299 (UAP,Version=v10.0.16299)	Microsoft.Toolkit.Uwp.UI.Controls.Design	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Design\Microsoft.Toolkit.Uwp.UI.Controls.Design.csproj	1	
Error	NU1201	Project Microsoft.Toolkit.Uwp.UI.Controls is not compatible with net452 (.NETFramework,Version=v4.5.2) / win-x86. Project Microsoft.Toolkit.Uwp.UI.Controls supports: uap10.0.16299 (UAP,Version=v10.0.16299)	Microsoft.Toolkit.Uwp.UI.Controls.Design	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Design\Microsoft.Toolkit.Uwp.UI.Controls.Design.csproj	1	
Error	NU1201	Project Microsoft.Toolkit.Uwp.UI.Controls is not compatible with net452 (.NETFramework,Version=v4.5.2) / win-x64. Project Microsoft.Toolkit.Uwp.UI.Controls supports: uap10.0.16299 (UAP,Version=v10.0.16299)	Microsoft.Toolkit.Uwp.UI.Controls.Design	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Design\Microsoft.Toolkit.Uwp.UI.Controls.Design.csproj	1	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.PlatformSpecificAnalyzer	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Parsers	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.Notifications	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.Notifications	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.UI	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.Connectivity	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.UI.Animations	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Services	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Services	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Services	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.DeveloperTools	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.Services	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.UI.Controls.DataGrid	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit\bin\Debug\netstandard2.0\Microsoft.Toolkit.dll'	Microsoft.Toolkit.Uwp.UI.Controls	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls\Microsoft.Toolkit.Uwp.UI.Controls.csproj	1	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Parsers\bin\Debug\netstandard2.0\Microsoft.Toolkit.Parsers.dll'	Microsoft.Toolkit.Uwp.UI.Controls	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls\Microsoft.Toolkit.Uwp.UI.Controls.csproj	1	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.dll'	Microsoft.Toolkit.Uwp.UI.Controls	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls\Microsoft.Toolkit.Uwp.UI.Controls.csproj	1	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Animations\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Animations.dll'	Microsoft.Toolkit.Uwp.UI.Controls	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls\Microsoft.Toolkit.Uwp.UI.Controls.csproj	1	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.dll'	Microsoft.Toolkit.Uwp.UI.Controls	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls\Microsoft.Toolkit.Uwp.UI.Controls.csproj	1	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.UI.Controls	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit\bin\Debug\netstandard2.0\Microsoft.Toolkit.dll'	Microsoft.Toolkit.Uwp.UI.Controls.Graph	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Graph\Microsoft.Toolkit.Uwp.UI.Controls.Graph.csproj	1	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Parsers\bin\Debug\netstandard2.0\Microsoft.Toolkit.Parsers.dll'	Microsoft.Toolkit.Uwp.UI.Controls.Graph	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Graph\Microsoft.Toolkit.Uwp.UI.Controls.Graph.csproj	1	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Services\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Services.dll'	Microsoft.Toolkit.Uwp.UI.Controls.Graph	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Graph\Microsoft.Toolkit.Uwp.UI.Controls.Graph.csproj	1	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.dll'	Microsoft.Toolkit.Uwp.UI.Controls.Graph	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Graph\Microsoft.Toolkit.Uwp.UI.Controls.Graph.csproj	1	
Error	WMC1006	Cannot resolve Assembly or Windows Metadata file 'E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI\bin\Debug\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.dll'	Microsoft.Toolkit.Uwp.UI.Controls.Graph	E:\Experimental\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.Graph\Microsoft.Toolkit.Uwp.UI.Controls.Graph.csproj	1	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.UI.Controls.Graph	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Microsoft.Toolkit.Uwp.Notifications.JavaScript	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	
Error	MSB4018	The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
LibGit2Sharp.NotFoundException: object not found - no match for id (20403dba3dd125ea155931f7a7469d595ad8088f)
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_revwalk_next(RevWalkerHandle walker)
   at LibGit2Sharp.CommitLog.CommitEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Nerdbank.GitVersioning.VersionOracle..ctor(String projectDirectory, Repository repo, Commit head, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.VersionOracle.Create(String projectDirectory, String gitRepoDirectory, ICloudBuild cloudBuild, Nullable`1 overrideBuildNumberOffset, String projectPathRelativeToGitRepoRoot)
   at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
   at MSBuildExtensionTask.ContextAwareTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	UnitTests.Notifications.NetCore	C:\Users\Encryptor\.nuget\packages\nerdbank.gitversioning\2.1.65\build\Nerdbank.GitVersioning.targets	63	

```